src/CommunityToolkit.Maui.Core/Views/Expander/PlatformView/MauiExpanderExtensions.shared.cs

```diff
@@ -0,0 +1,69 @@
+using CommunityToolkit.Maui.Core.Views;
+using Microsoft.Maui.Platform;
+#if WINDOWS
+using MauiExpander = Microsoft.UI.Xaml.Controls.Expander;
+#endif
+namespace CommunityToolkit.Maui.Core.Extensions;
+
+/// <summary>
+/// Extension methods to support <see cref="IExpander"/>
+/// </summary>
+public static partial class MauiExpanderExtensions
+{
+#if ANDROID || IOS || MACCATALYST || WINDOWS
```

Comment:
Should Tizen be here also?

I also note this question is out of curiosity if there is a reason other than unfamiliarity and/or testing difficulty. I don't have competence to consider other stuff here.

Reply:
No, we do not support Tizen right now